### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by arapulido**
* Point to our Securing Kubeapps guide in every install reference
* Remove the install.md file, as it doesn't have useful information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ helm repo add bitnami https://charts.bitnami.com/bitnami
 helm install --name kubeapps --namespace kubeapps bitnami/kubeapps
 ```
 
+> **IMPORTANT** This assumes a non-secured Helm installation in your cluster. We don't recommend this for a production environment. Please, follow [the Kubeapps guide on how to secure Helm and Kubeapps for production](docs/securing-kubeapps.md) to secure your Helm and Kubeapps installation.
```
Comment (prydonius) [inline README.md]: I think this is a bit easier to read and more succinct:

> This assumes an insecure Helm installation, which is not recommended in production. See the documentation to [learn how to secure Helm and Kubeapps in production](docs/securing-kubeapps.md).

File: chart/kubeapps/README.md [hunk 2]
```diff
@@ -40,6 +40,8 @@ $ helm repo add bitnami https://charts.bitnami.com/bitnami
 $ helm install --name kubeapps --namespace kubeapps bitnami/kubeapps
 ```
 
+> **IMPORTANT** This assumes a non-secured Helm installation in your cluster. We don't recommend this for a production environment. Please, follow [the Kubeapps guide on how to secure Helm and Kubeapps for production](https://github.com/kubeapps/kubeapps/blob/master/docs/user/securing-kubeapps.md) to secure your Helm and Kubeapps installation.
```
Comment (prydonius) [inline chart/kubeapps/README.md]: I think this is a bit easier to read and more succinct:

> This assumes an insecure Helm installation, which is not recommended in production. See the documentation to [learn how to secure Helm and Kubeapps in production](docs/securing-kubeapps.md).

File: docs/user/getting-started.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ This guide will walk you through the process of deploying Kubeapps for your clus
 
 ## Prerequisites
 
-Kubeapps assumes a working Kubernetes cluster (v1.8+) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed and configured to talk to your Kubernetes cluster. Kubeapps binaries are available for Linux, OS X and Windows, and Kubeapps has been tested with Azure Kubernetes Service (AKS), Google Kubernetes Engine (GKE), `minikube` and Docker for Desktop Kubernetes. Kubeapps works on RBAC-enabled clusters and this configuration is encouraged for a more secure install.
+Kubeapps assumes a working Kubernetes cluster (v1.8+) and [`Helm`](https://helm.sh/) (2.9.1+) installed in your cluster and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed and configured to talk to your Kubernetes cluster. Kubeapps has been tested with Azure Kubernetes Service (AKS), Google Kubernetes Engine (GKE), `minikube` and Docker for Desktop Kubernetes. Kubeapps works on RBAC-enabled clusters and this configuration is encouraged for a more secure install.
```
Comment (prydonius) [inline docs/user/getting-started.md]: remove the "and" between "Kubernetes cluster and Helm" and replace with a comma

File: docs/user/getting-started.md [hunk 4]
```diff
@@ -15,6 +15,8 @@ helm repo add bitnami https://charts.bitnami.com/bitnami
 helm install --name kubeapps --namespace kubeapps bitnami/kubeapps
 ```
 
+> **IMPORTANT** This assumes a non-secured Helm installation in your cluster. We don't recommend this for a production environment. Please, follow [the Kubeapps guide on how to secure Helm and Kubeapps for production](securing-kubeapps.md) to secure your Helm and Kubeapps installation.
```
Comment (prydonius) [inline docs/user/getting-started.md]: I think this is a bit easier to read and more succinct:

> This assumes an insecure Helm installation, which is not recommended in production. See the documentation to [learn how to secure Helm and Kubeapps in production](docs/securing-kubeapps.md).

**Comment by arapulido** [comment]
@prydonius ptal

File: README.md [hunk 5]
```diff
@@ -26,6 +26,8 @@ helm repo add bitnami https://charts.bitnami.com/bitnami
 helm install --name kubeapps --namespace kubeapps bitnami/kubeapps
 ```
 
+> **IMPORTANT** This assumes an insecured Helm installation, which is not recommended in production. See [the documentation to learn how to secure Helm and Kubeapps in production](securing-kubeapps.md).
```
Comment (prydonius) [inline README.md]: s/insecured/insecure

File: chart/kubeapps/README.md [hunk 6]
```diff
@@ -40,6 +40,8 @@ $ helm repo add bitnami https://charts.bitnami.com/bitnami
 $ helm install --name kubeapps --namespace kubeapps bitnami/kubeapps
 ```
 
+> **IMPORTANT** This assumes an insecured Helm installation, which is not recommended in production. See [the documentation to learn how to secure Helm and Kubeapps in production](securing-kubeapps.md).
```
Comment (prydonius) [inline chart/kubeapps/README.md]: s/insecured/insecure

File: docs/user/getting-started.md [hunk 7]
```diff
@@ -15,6 +15,8 @@ helm repo add bitnami https://charts.bitnami.com/bitnami
 helm install --name kubeapps --namespace kubeapps bitnami/kubeapps
 ```
 
+> **IMPORTANT** This assumes an insecured Helm installation, which is not recommended in production. See [the documentation to learn how to secure Helm and Kubeapps in production](securing-kubeapps.md).
```
Comment (prydonius) [inline docs/user/getting-started.md]: s/insecured/insecure

**Pull request merged**